main.cpp

```diff
@@ -348,6 +348,97 @@ void convertWEAP(SkyblivionConverter &converter) {
 
 }
 
+void addSpeakAsNpcs(SkyblivionConverter &converter, Collection &skyrimCollection) {
+	std::string metadataFile = converter.ROOT_BUILD_PATH() + "Metadata";
```

Comment:
We'll have to extract the metadata parsing part into more generic class , but for now i suppose its okay